core/whistle_number_manager-1.0.0/src/wh_port_request.erl

```diff
@@ -120,6 +125,8 @@ maybe_transition(PortReq, ?PORT_SCHEDULED) ->
 maybe_transition(PortReq, ?PORT_COMPLETE) ->
     transition_to_complete(PortReq);
 maybe_transition(PortReq, ?PORT_REJECT) ->
+    transition_to_rejected(PortReq);
+maybe_transition(PortReq, ?PORT_CANCELED) ->
```

Comment:
Boo copy/paste without testing! This clause calls `transition_to_rejected` when it should be `transition_to_canceled`